File: Exercicio-1/calcule-raiz-segundo2.cpp
#include <stdio.h>
#include <stdlib.h>
int main(){
	
	float a, b, c, delta;
	printf("===================================================================================\n");
	printf("                                 Atividade\n");
	printf("===================================================================================\n");
	
	printf("Informe o valor de [(a] :  ");
	scanf("%f", &a);
	if (a == 0){
	    printf("A equacao nao eh do segundo grau. \n Operacao Finalizada.\n");
	}else{
		printf("Informe o valor de [(b] :  ");
		scanf("%f", &b);
		printf("Informe o valor de [(c] :  ");
		scanf("%f", &c);
    }
	delta = (b*b)-4*a*c;
	printf("===================================================================================\n");
	if (delta < 0){
	    printf("[Alerta: Delta calculado eh negativo, a equacao nao possui raizes reais.]\n");
	}else
	    if (delta == 0){
	        printf("[Alerta: Delta calculado eh igual a zero a equacao possui apenas uma raiz real.]\n");
	    }else
	        if (delta >= 0){
	           printf("[Alerta: A equacao possui duas raizes reais.]\n");
	    }    
	printf("===================================================================================\n");
	system("PAUSE");
	return(0);
}

File: Code_Python/Exercicio-04/Calcule-raiz-segundo2.py
#Questão 9

print('='*50)
print(" " * 19, "Atividade")
print('='*50)

a = float(input("Informe o valor de [(a] :  "))
if a == 0:
    print("A equação não é do segundo grau. \n Operação Finalizada.")
else:
    b = float(input("Informe o valor de [(b] :  "))
    c = float(input("Informe o valor de [(c] :  "))
    delta = (b*b)-4*a*c
    print('=' * 50)
    if delta < 0:
        print("[Alerta: Delta calculado é negativo, a equação não possui raizes reais.]")
    elif delta == 0:
        print("[Alerta: Delta calculado é igual a zero a equação possui apenas uma raiz real.]")
    elif delta >= 0:
        print("[Alerta: A equação possui duas raizes reais.]")

print('='*50)